pkg/volume/azure_file/azure_util.go

```diff
@@ -18,13 +18,21 @@ package azure_file
 
 import (
 	"fmt"
+	"strings"
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/pkg/volume"
 )
 
+const (
+	fileModeName    = "file_mode"
```

Comment:
a bit paranoid but `file_mode=` is safer to me (and `dir_mode=`) too

Reply:
fileMode is better, underscore is not good go var naming convetion